### DISCUSSION TITLE
VM Execution error

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1568] [Q&A] [answered]
When I try to get latest price, This error comes up.
![Screenshot from 2022-07-04 13-27-29](https://user-images.githubusercontent.com/77268333/177109082-ba0db52c-4169-40be-8250-f4d9cb13e3a9.png)

**Comment by cromewar** [comment, marked as answer]
Hello @Akhil-franko 
Remember the pricefeed Address depends a lot on the network you are working, so please be sure to select the proper network on metamask in order to proceed.

**Reply by akhilparakka** [reply]
Thank You. Got It.
